firmware/firmware.h

```diff
@@ -65,7 +65,7 @@ void flash_disable();
 #define BRIDGE_USB 0
 #define BRIDGE_PORT_A 1
 #define BRIDGE_PORT_B 2
-#define BRIDGE_BUF_SIZE 256
+#define BRIDGE_BUF_SIZE 255
```

Comment:
It's possibly responsible for https://github.com/tessel/t2-firmware/issues/166, because if you fill the reply buffer to 256 bytes, and pass it to the bridge functions, it will wrap to zero, which the bridge will interpret as "no message to send" and the port will get stuck waiting for it to complete.